python/podio_schema_evolution.py

```diff
@@ -327,6 +425,33 @@ def heuristics(self):
         added_members = [change for change in schema_changes if isinstance(change, AddedMember)]
         self.heuristics_members(added_members, dropped_members, schema_changes)
 
+        for vmc in (c for c in schema_changes if isinstance(c, AddedVectorMember)):
+            self.errors.append(
+                f"Forbidden schema change in '{vmc.klassname}': Added vector member '{vmc.member}'"
+            )
+        for vmc in (c for c in schema_changes if isinstance(c, DroppedVectorMember)):
+            self.errors.append(
+                f"Forbidden schema change in '{vmc.klassname}': Added vector member '{vmc.member}'"
```

Comment:
```suggestion
                f"Forbidden schema change in '{vmc.klassname}': Dropped vector member '{vmc.member}'"
```
?
What does vmc stand for? vector member change?
Why is this forbidden?

Reply:
Forbidden in this case is to be interpreted as *not implemented*. If users make this schema change trying to read old files will most likely crash with some sort of segmentation violation.